### DISCUSSION TITLE
Fretboard Diagrams

### DISCUSSION THREAD

**Discussion by ObdulX** [musescore/MuseScore#12781] [General]
Hi,
for educational purposes I loved that in Musescore 3 I can make the Fretbordboard Diagrams have up to 22 frets. (For showing Scale connections etc).
In Musescore 4 there are only 6 frets possible. This is a step back. What a pity.
Hopefully in the release version this will be changed.
Anyway great job. Thanks to all the programmer

**Comment by nevermore2k** [comment]
I need them too for teaching purposes.  I recently started to use Finale again, cause of that ;) .... Finale...

**Reply by ShibuyaCyana** [reply]
It's on the way! It's possible again in 4.2 beta:
![image](https://github.com/musescore/MuseScore/assets/39703247/0f122ded-6161-4a6a-941d-34291bc337fe)

**Reply by nevermore2k** [reply]
nicu

**Comment by ObdulX** [comment]
This would be fine. :)Am 04.12.2023 um 23:24 schrieb nevermore2k ***@***.***>:﻿
nicu

**Comment by jpygbp** [comment]
Are there any plans to attach the fretboard diagram to the chords that are translated?

Thanks,
